browser/src/control/jsdialog/Widget.IconView.ts

```diff
@@ -116,7 +116,7 @@ function _iconViewEntry(
 
 	if (!disabled) {
 		const singleClick = parentData.singleclickactivate === true;
-		$(entryContainer).click(function () {
+		$(entryContainer).on('click', function (event: JQuery.ClickEvent) {
```

Comment:
we port code to non-JQuery, maybe we can do the same here? to not rework it again?